lutris/gui/lutriswindow.py

```diff
@@ -66,6 +66,8 @@ def __init__(self, application, **kwargs):
         width = int(settings.read_setting('width') or 800)
         height = int(settings.read_setting('height') or 600)
         self.window_size = (width, height)
+        self.maximized  = settings.read_setting('maximized') == 'True'
```

Comment:
`True` as a string? Is this correct? (This might be correct if it's some (GUI) framework thing, but in general Python `True` is its [its own value](https://docs.python.org/3/library/stdtypes.html#boolean-values).)

(Also, there should only be one space before the `=`.)